mlx/traceability.py

```diff
@@ -718,6 +754,19 @@ def process_item_nodes(app, doctree, fromdocname):
 
         node.replace_self(new_node)
 
+    # Processing of the item-link items. They get added as additional relationships
```

Comment:
move this up in the fuction to right below the self_test(), line 583

all relations need to be known before rendering item-matrix, etc